Pipeline/CreateUCDComponentVersion/dbb-ucd-packaging.groovy

```diff
@@ -495,20 +626,38 @@ if (!properties.preview.toBoolean()){
 }
 
 /**
- * parse data set name and member name
- * @param fullname e.g. BLD.LOAD(PGM1)
- * @return e.g. (BLD.LOAD, PGM1)
+ * return true if the file is a file stored on zFS, false otherwise
  */
-def getDatasetName(String fullname){
-	def ds,member;
-	def elements =  fullname.split("[\\(\\)]");
-	ds = elements[0];
-	member = elements.size()>1? elements[1] : "";
-	return [ds, member];
+def isZFSFile(String name) {
+    Path path = Paths.get(name);
+    return Files.exists(path);
 }
 
 /**
+ * Extract the directory and the file 
+ * from the fullname of a zFS file
+ * For instance: /var/test/file.txt  --> [/var/test, file.txt]
  */
+def extractDirectoryAndFile(String fullname) {
+    Path filePath = Paths.get(fullname);
+    String file = filePath.getFileName().toString();
+    String directory = filePath.getParent();
+    return [directory, file];
+}
+
+/**
+ * Parse the fullname of a qualified dataset and member name
+ * Returns its dataset name and member name)  
+ * For instance: BLD.LOAD(PGM1)     --> [BLD.LOAD, PGM1]
+ */
+def getContainerAndFile(String fullname) {
```

Comment:
Would prefer `getDatasetAndMember` as a name of the method. It is closer to what it does.

Reply:
Done - I was trying to define a generic function for zFS files and dataset members, but finally did it in two different functions, and forgot to change the name of this one.